scalafmt-tests/src/test/resources/scalajs/Apply.stat

```diff
@@ -181,3 +181,295 @@ object a {
+}
+>>>
+object a {
+  new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name, 0,
+              len, openAngleBracketOK = false)))
+}
+<<< #2079 avoid nested indent, oneline; 2
+binPack.preset = true
+binPack.unsafeCallSite = oneline
+continuationIndent.callSite = 4
+===
+object a {
+  new SimpleMethodName(new SimpleMethodName(
+    validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false)))
+}
+>>>
+object a {
+  new SimpleMethodName(new SimpleMethodName(
+          validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false)))
+}
+<<< #2079 avoid nested indent, always; 3
+binPack.preset = true
+binPack.unsafeCallSite = always
+continuationIndent.callSite = 4
+===
+object a {
+  foo(
+    new SimpleMethodName(new SimpleMethodName(
+      validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false))),
+    new SimpleMethodName(new SimpleMethodName(
+      validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false))))
+}
+>>>
+object a {
+  foo(new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name,
+                  0, len, openAngleBracketOK = false))),
+      new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name,
+                  0, len, openAngleBracketOK = false))))
+}
+<<< #2079 avoid nested indent, oneline; 3
+binPack.preset = true
+binPack.unsafeCallSite = oneline
+continuationIndent.callSite = 4
+===
+object a {
+  foo(
+    new SimpleMethodName(new SimpleMethodName(
+      validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false))),
+    new SimpleMethodName(new SimpleMethodName(
+      validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false))))
+}
+>>>
+object a {
+  foo(
+      new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name,
+                  0, len, openAngleBracketOK = false))),
+      new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name,
+                  0, len, openAngleBracketOK = false))))
+}
+<<< #2079 avoid nested indent, always; 4
+binPack.preset = true
+binPack.unsafeCallSite = always
+continuationIndent.callSite = 4
+===
+object a {
+  new SimpleMethodName(new SimpleMethodName(new SimpleMethodName(
+    validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false)))).foo
+}
+>>>
+object a {
+  new SimpleMethodName(
+      new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name,
+                  0, len, openAngleBracketOK = false)))).foo
+}
+<<< #2079 avoid nested indent, oneline; 4
+binPack.preset = true
+binPack.unsafeCallSite = oneline
+continuationIndent.callSite = 4
+===
+object a {
+  new SimpleMethodName(new SimpleMethodName(new SimpleMethodName(
+    validateSimpleEncodedName(name, 0, len, openAngleBracketOK = false)))).foo
+}
+>>>
+object a {
+  new SimpleMethodName(
+      new SimpleMethodName(new SimpleMethodName(validateSimpleEncodedName(name,
+                  0, len, openAngleBracketOK = false)))
```

Comment:
This indentation seems excessive, shouldn't it just be 4? Looks like a lot of other test cases have double indentation. This even seem 3x (12 spaces)

Reply:
it only looks strange with binpacking. each open paren adds an indentation. there is an indent once setting, for those who like this type of [un]formatting.

Reply:
Ach, it makes sense! Thanks for explaining.